README.md

```diff
@@ -180,91 +184,107 @@ The binary executables are saved in the out directory.
 #### Example Usage
```

Comment:
I think we are duplicating the section. we should simplify it to just one repetition. or, instead, provide more detailed information

Reply:
Good point. I will just stick to only one repetition.

Reply:
In my newest commit I deleted the "Example Usage" section. Maybe this section could actually be used to show an example logstash exporter usage scenario? I think it might be helpful for any newcomer.

Reply:
i think that running the application is documented enough, but if you want you can create examples dir
personally i dont think it is needed, people looking for this tool usually know what they are doing :p